<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculatrice</class>
 <widget class="QWidget" name="calculatrice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="spinBoxNombre1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxOperation">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>86</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>/</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBoxNombre2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>48</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEgal">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResultat">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
